File: Deafios em C#/1114 - Senha Fixa.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1114 {

    static void Main(string[] args) { 

        string senha="2002";  
        string senhauser; 

        do{                    
            senhauser=Console.ReadLine();   

            if(senha != senhauser){

                Console.WriteLine("Senha Invalida"); 
            }

        }while(senha != senhauser);               

        Console.WriteLine("Acesso Permitido"); 

    }

}

File: Desafios em JavaScript/1114 - Senha Fixa.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var senha="2002";  
var senhauser; 

do{                    
    senhauser=lines.shift();

    if(senha !== senhauser){

        console.log("Senha Invalida"); 
    }

}while(senha !== senhauser);               

console.log("Acesso Permitido"); 